src/Compiler/Checking/CheckPatterns.fs

```diff
@@ -35,6 +35,24 @@ type cenv = TcFileState
 // Helpers that should be elsewhere
 //-------------------------------------------------------------------------
 
+// CAUTION: This function operates over the untyped tree, so should be used only when absolutely necessary. It doesn't verify assembly origine nor does it respect type aliases.
+let inline findSynAttribute (attrName: string) (synAttrs: SynAttributes) =
+    let attributesToSearch =
+        if attrName.EndsWith("Attribute") then
+            set [attrName; attrName.Substring(0, attrName.Length - 9).ToString()]
```

Comment:
.ToString() is still redundant here, right?

Reply:
Sigh...yes. Once again, I removed the ToSpan, but not ToString